src/test/java/core/basesyntax/operationswithfile/FileWriterImplTest.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.operationswithfile;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FileWriterImplTest {
+    private static final String TRUE_PATH = "src/test/java/resources/file.csv";
+    private static final String FALSE_PATH = "srv/test/java/core/basesyntax/resources/newFile.csv";
+    private static final Map<String, Integer> balance = new HashMap<>();
+    private static final FileWriter fileWriter = new FileWriterImpl();
+
+    @BeforeClass
+    public static void addToBalanceMap() {
+        balance.put("banana", 50);
+        balance.put("apple",20);
+    }
+
+    @Test
+    public void testWriteToFileWithTruePath_ok() {
```

Comment:
this test will always return true! You must compare some result with the expected data.

Reply:
added an additional condition